curriculum/challenges/english/25-front-end-development/quiz-javascript-variables-and-data-types/66edc25ae5ea80bf6f785552.md

```diff
@@ -127,23 +127,23 @@ What is the difference between `let` and `const` variable declarations in JavaSc
 
 #### --text--
 
-What is the difference between local and global scope in JavaScript?
+
```

Comment:
Maybe a question pertaining all three different scopes?

Reply:
scopes are not mentioned in the lectures